proxy/interceptor.go

```diff
@@ -3298,6 +3450,19 @@ func visitFailures(ctx *VisitFailuresContext, options *VisitFailuresOptions, obj
 				return err
 			}
 
+		case *workflow.WorkflowExecutionConfig:
+			if o == nil {
+				continue
+			}
+			ctx.Parent = o
+			if err := visitFailures(
+				ctx,
+				options,
+				o.GetUserMetadata(),
```

Comment:
`GetUserMetadata` doesn't contain a failure so why would we need to search it?

Reply:
It looks like maybe some of these are still reporting as being needed to search because they have payloads rather than failures, I'll see if that's it.

Reply:
Ok these were fixed by removing the hardcoded check for payloads in `walk`